source/MRMesh/MRPolylineTrimWithPlane.cpp

```diff
@@ -48,11 +48,13 @@ namespace MR
                 }              
                 e0 = polyline.topology.next( e0.sym() );
             }
+
+            cutSegments.push_back( { polyline.topology.org( e ), polyline.topology.org( e0.sym() ) } );
```

Comment:
mb resize it before, and also mb not fill it if we don't do cut (it should be optional argument)